.github/workflows/workflow_call.unit-test.yml

```diff
@@ -41,9 +41,8 @@ jobs:
           - "1.24"
         os:
           - ubuntu-latest
-          # TODO: Enable tests for relevant OSes.
-          # - macos-latest
-          # - windows-latest
+          - macos-latest
```

Comment:
Tests must be run with GNU make on macOS. Install gmake with homebrew and run tests using the gmake command. 

Make sure to do that on macOS only.